common/edicts/iisc_edicts.txt

```diff
@@ -31,7 +31,7 @@ iisc_modmenu_edict = {
 	}
 	potential = {
 		NOT = {
-			is_ai = yes
+			has_global_flag = dmm_installed
```

Comment:
Maybe use a NOR and keep the is_ai = yes?
That way it won't be used by the ai even if dmm is not installed.